src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    private static final char SPECIFIED_CHARACTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> strings;
+        String[] result;
+
+        try {
+            strings = Files.readAllLines(file.toPath());
```

Comment:
```suggestion
            strings = Files.readAllLines(Path.of(fileName));
```
this way you won;t need to declare file variable